pokete_data/mapstations.py

```diff
@@ -299,6 +300,21 @@
             "y": 1
         }
     },
+    "playmap_39": {
+        "gen": {
+            "additionals": ["playmap_41", "playmap_42", "playmap_43",
+                            "playmap_44", "playmap_45", "playmap_46",
+                            "playmap_47", "playmap_48"],
+            "width": 2,
+            "height": 2,
+            "desc": "A Great city of Agrawos, agracultural and cultural center of the whole region. It's famous for its' great Pokete-Arena and its' master trainer. Check out the MowCow-Burger restaurant, which offers the best, juiciest and most delicious Mowcow-burgers, cut from the happiest and most delicious Mowcows anywhere to find!",
```

Comment:
This line is way too long, also there are two small typos:

```suggestion
            "desc": "The great city of Agrawos, agricultural and cultural "
                    "center of the whole region. It's famous for its great "
                    "Pokete-Arena and its master trainer. Check out the "
                    "MowCow-Burger restaurant, which offers the best, "
                    "juiciest and most delicious Mowcow-burgers, cut from the "
                    "happiest and most delicious Mowcows anywhere to find!", 
```

Please also be clear about the pokete being called `MowCow` or `Mowcow` in the texts!

Reply:
Like the Pokete Mowcow is called Mowcow, but the brandname of the burgerrestaurant in MowCow-Burger.